src/tools/interop/interop.cpp

```diff
@@ -596,6 +602,15 @@ class InteropConnection {
         }
         return false;
     }
+    bool GetResumptionTicket(const uint8_t*& Ticket, uint32_t& TicketLength) {
+        if (!WaitForTicket() || ResumptionTicket == nullptr) {
```

Comment:
` || ResumptionTicket == nullptr` should be unnecessary, no?

Reply:
If this gets called twice, the second call will return null. Why not just fail the call?